src/Components/ServiceScene/Breakdowns/LayoutSwitcher.tsx

```diff
@@ -27,6 +32,24 @@ export class LayoutSwitcher extends SceneObjectBase<LayoutSwitcherState> {
     this.addActivationHandler(this.onActivate.bind(this));
   }
 
+  public isTopLevelLayoutType = () => {
+    // Top layouts do not have 'single' layout type
+    const isTopLevel = this.state.options.every((o) => o.value !== LayoutTypeEnum.single);
+    return isTopLevel;
+  };
+
+  public updateLayout = () => {
+    const layout = getSceneLayout();
+    if (layout) {
+      if (layout === LayoutTypeEnum.single && this.isTopLevelLayoutType()) {
+        // top level layouts do not have single layout type default to grid
+        this.setState({ active: 'grid' });
+      } else {
+        this.setState({ active: layout as LayoutType });
```

Comment:
Let's drop the type assertion and instead narrow the layout type, 
`active: (layout === LayoutTypeEnum.grid || layout === LayoutTypeEnum.rows) ? layout :  LayoutTypeEnum.grid`